documentation-site/pages/blog/responsive-web/index.mdx

```diff
@@ -0,0 +1,248 @@
+
+```js
+<head> // or <Helmet>
+  <meta
+    name="viewport"
+    content="width=device-width, initial-scale=1, shrink-to-fit=no"
+  />
+</head>
+```
+
+This meta tag was originally introduced by [Apple](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/UsingtheViewport/UsingtheViewport.html) for iOS Safari, but has since become an industry standard for controlling the viewport’s size and scale.
+
+- **`width=device-width`** sets the width of the page equal to the width of the device or window. E.g. if your screen has a width of 375px, then `width: 100%` will by default set width to 375px.
+- **`initial-scale=1`** tells the browser to start at the 1:1 zoom level.
+- **`shrink-to-fit=no`** further prevents the page from scaling down in the event that you have elements on the page that overflow (either intentionally or unintentionally) the viewport bounds.
+
+## Custom breakpoints
+
+In many cases, you would want to override Base Web’s breakpoints with your own. The default breakpoints for Base Web are:
+
+- **`small: 320`**
+- **`medium: 600`**
+- **`large: 1280`**
+
+To override these, we recommend creating your own mapping, as well as generating the respective `min-width` media query strings using `getMediaQuery()` ([v6.19.0](https://github.com/uber-web/baseui/releases/tag/v6.19.0)+), so we could target different screen widths for styling:
+
+```js
+const MyBreakpoints = {
+  small: 769,
+  medium: 1024,
+  large: 1216,
+};
+
+const ResponsiveTheme = Object.keys(MyBreakpoints).reduce(
+  (acc, key) => {
+    acc.breakpoints[key] = MyBreakpoints[key];
+    acc.media[key] = getMediaQuery({
+      'min-width': `${MyBreakpoints[key]}px`,
+    });
+    return acc;
+  },
+  {
+    breakpoints: {},
+    media: {},
+  },
+);
+
+export const MyTheme = {
+  ...LightTheme,
+  ...ResponsiveTheme,
+};
+```
+
+This sets the following values inside `MyTheme`:
+
+```json
+{
+  "breakpoints": {
+    "small": 769
+    "medium": 1024,
+    "large": 1216,
+  },
+  "media": {
+    "small": "@media screen and (min-width: 769px)"
+    "medium": "@media screen and (min-width: 1024px)",
+    "large": "@media screen and (min-width: 1216px)",
+  }
+}
+```
+
+Then you can use `MyTheme` throughout your app using Base Web’s [`ThemeProvider`](/theming/custom-themes#the-themeprovider).
+
+## Styling components using `themedStyled()`
+
+Now that our theme is set up, we can start using it to create styled components using `themeStyled()`. Using the media queries we created above, we will write the mobile styles first, and add desktop styles under the media query key:
```

Comment:
I feel like you can expand more on the idea of styling for mobile first — it's not intuitive enough to grasp just by a mention and some code block imo. What are the pros and cons of doing mobile-first styling? Why is it important?

It will also make it easier to understand the Block API down below, "mobile first" will mean that the array reads small screen -> larger screen.

Reply:
Good catch! Fixed